src/utils/daos_hdlr.h

```diff
@@ -70,11 +93,18 @@ struct cmd_args_s {
 	enum pool_op		p_op;		/* pool sub-command */
 	enum cont_op		c_op;		/* cont sub-command */
 	enum obj_op		o_op;		/* obj sub-command */
+	enum fs_op		fs_op;		/* filesystem sub-command */
 	char			*sysname;	/* --sys-name or --sys */
 	uuid_t			p_uuid;		/* --pool */
+	uuid_t			src_p_uuid;	/* --src_pool */
```

Comment:
@mchaarawi well I suppose I fill them in after parsing the uuids. I did that because some of the other options have the uuid_t variable stored in the args struct, but I don't think it really has to be stored there.